src/robusta/integrations/webex/sender.py

```diff
@@ -0,0 +1,253 @@
+                    container["items"][0]["columns"].append(
+                        {
+                            "type": "Column",
+                            "width": "stretch",
+                            "items": [
+                                {"type": "TextBlock", "text": header, "wrap": "true"}
+                            ],
+                        }
+                    )
+                #seperating each row to add below headers of column
+                rows = block.render_rows()
+                for row in rows:
+                    row_json = {"type": "ColumnSet", "columns": []}
+                    for text in row:
+                        row_json["columns"].append(
+                            {
+                                "type": "Column",
+                                "width": "stretch",
+                                "items": [
+                                    {"type": "TextBlock", "text": text, "wrap": "true"}
+                                ],
+                            }
+                        )
+                    container["items"].append(row_json)
+                adaptive_card["body"].append(container)
+
+        return adaptive_card
+
+    def _seperate_blocks(self, finding: Finding, platform_enabled: bool):
+        table_blocks: List[TableBlock] = []
+        file_blocks: List[FileBlock] = []
+        description = None
+
+        message_content = self._create_message_content(
+            finding, platform_enabled, self.cluster_name
+        )
+
+        blocks = [MarkdownBlock(text=f"*Source:* _{self.cluster_name}_\n\n")]
+
+        # Seperate blocks into *Other* Blocks, TableBlocks and FileBlocks
+        for enrichment in finding.enrichments:
+            blocks.extend(
+                [
+                    block
+                    for block in enrichment.blocks
+                    if self.__is_webex_text_block(block)
+                ]
+            )
+            table_blocks.extend(
+                [block for block in enrichment.blocks if isinstance(block, TableBlock)]
+            )
+            file_blocks.extend(
+                [block for block in enrichment.blocks if isinstance(block, FileBlock)]
+            )
+
+        # first add finding description block
+        if finding.description:
+            if table_blocks:
+                description = finding.description
+            else:
+                blocks.append(MarkdownBlock(finding.description))
+
+        # Convert *Other* blocks to markdown
+        for block in blocks:
+            block_text = Transformer.to_standard_markdown([block])
+            if (
+                len(block_text) + len(message_content) >= MAX_BLOCK_CHARS
+            ):  # webex message size limit
+                break
+            message_content += block_text + "\n"
+
+        return message_content, table_blocks, file_blocks, description
+
+    def _create_pdf(self, file_blocks: List[FileBlock]):
```

Comment:
I tested the sink, and the PDF workaround does not seem to work (I've added an image). To be honest, while the idea is clever, it is also very fragile. Also complicates the code.  

I suggest we lose that part and send all the files separately with different messages or stop at the limit.  Hopefully, they will add multifile support soon.

I didn't take it lightly but I think it will be a better approach 





![image](https://user-images.githubusercontent.com/20162469/195172754-10c8b99e-4ec9-4616-93bc-86c35a6c8ed4.png)

Reply:
I will update it and send the files separately.

Reply:
Updated. Let me know if its ok or anything needs to be changed.